doc/classes/OS.xml

```diff
@@ -408,6 +408,15 @@
 				[b]Note:[/b] On Web platforms, it is still possible to determine the host platform's OS with feature tags. See [method has_feature].
 			</description>
 		</method>
+		<method name="get_process_exit_code" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="pid" type="int" />
+			<description>
+				Returns the exit code of a spawned process once it has finished running (see [method is_process_running]).
+				Returns [code]-1[/code] if the [param pid] is not a PID of a spawned child process, the process is still running, or the method is not implemented for the current platform.
+				[b]Note:[/b] This method is implemented on Android, iOS, Linux, macOS and Windows.
```

Comment:
Actually, I wouldn't list iOS, it is technically supported, but `execute` and other process related methods will work only on a jailbroken devices.

Reply:
The consensus among recent forum posts is that fork is not forbidden on iOS, [as long as any process you execute has the correct app-sandbox entitlements](https://forums.developer.apple.com/forums/thread/747499?answerId=781561022#781561022).
It's fine like it is: we don't need to mention support and we also don't need to forbid it. Just thought I'd mention.

Reply:
This is about the macOS sandbox (you can spawn processes with some restrictions). On iOS it's fully blocked.